### ISSUE TITLE
Val map getting lower and lower

### ISSUE THREAD

**Issue by YangChen1234567** [mit-han-lab/bevfusion#254]
Hi!
I'm trying to reproduce the results that was reported, so i load lidar-only det model to train the fusion model. But as training progress, the val map and nds value is getting lower and lower. In the first epoch, map = 0.62, but when epoch 7, map = 0.59.....
Could you please tell me why....... 
Help please.... TAT 555..

**Comment by kentang-mit** [comment]
Please try using a cosine annealing schedule with starting learning rate=2e-4 and train for 6 epochs. Loading pretrained LiDAR-only checkpoint is necessary.
